RELEASE.md

```diff
@@ -8,22 +8,16 @@
 
 *   `tf.function`:
 
-    *   tf.function now uses the Python inspect library directly for parsing the signature of the Python function it is decorated on.
-    *   This can break certain cases that were previously ignored where the signature is malformed, e.g.
-            *   Using functools.wraps on a function with different signature
-            *   Using functools.partial with an invalid tf.function input
-    *   tf.function now enforces input parameter names to be valid Python identifiers. Incompatible names are automatically sanitized similarly to existing SavedModel signature behavior.
+    *   `tf.function` now uses the Python inspect library directly for parsing the signature of the Python function it is decorated on.
+    *   This can break certain cases that were previously ignored where the signature is malformed, such as:
+        *   Using `functools.wraps` on a function with different signature
+        *   Using `functools.partial` with an invalid `tf.function` input
+    *   `tf.function` now enforces input parameter names to be valid Python identifiers. Incompatible names are automatically sanitized similarly to existing SavedModel signature behavior.
     *   Parameterless tf.functions are assumed to have an empty input_signature instead of an undefined one even if the input_signature is unspecified.
```

Comment:
```suggestion
    *   Parameterless `tf.function`s are assumed to have an empty `input_signature` instead of an undefined one even if the `input_signature` is unspecified.
```